### ISSUE TITLE
Add circleci template

### ISSUE THREAD

**Issue by ethanfrey** [confio/cosmwasm-template#7]
Let's include the `.circleci/config.yml` to do basic tests out of the box.

We can base it on the escrow example: https://github.com/confio/cosmwasm-examples/blob/master/.circleci/config.yml#L9-L44

**Comment by webmaster128** [comment]
Isn't this in here already? https://github.com/confio/cosmwasm-template/tree/master/.circleci It is just not running

**Comment by ethanfrey** [comment]
I thought to do it, and forgot that I ended up doing so.

It cannot run on this repo - the template is not valid code. But it should be a few clicks (in circleci) to enable testing on any new repo. No setup required. 

Actually, that should make it into the README

**Comment by webmaster128** [comment]
Okay, the file is copied 1:1 right now because otherwise it fails substituting stuff like `key: v4-cargo-cache-{{ arch }}-{{ checksum "Cargo.lock" }}`.

I tried the following patch


```diff
diff --git a/.circleci/config.yml b/.circleci/config.yml
index 6bb9e4a..93b8792 100644
--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -1,10 +1,11 @@
 version: 2.1
 
 jobs:
-  build:
+  {{project-name}}:
     docker:
       - image: rust:1.38
     steps:
+{% raw %}
       - checkout
       - run:
           name: Version information
@@ -35,3 +36,4 @@ jobs:
             - target/wasm32-unknown-unknown/release/build
             - target/wasm32-unknown-unknown/release/deps
           key: v4-cargo-cache-{{ arch }}-{{ checksum "Cargo.lock" }}
+{% endraw %}
diff --git a/cargo-generate.toml b/cargo-generate.toml
index 5ff09bf..8575a32 100644
--- a/cargo-generate.toml
+++ b/cargo-generate.toml
@@ -1,2 +1,2 @@
 [template]
-exclude = [".circleci/config.yml"]
\ No newline at end of file
+exclude = []
```

which fails when generating a project with with

```
$ cargo generate --git /Users/my/local/checkout/cosmwasm-template --name foobar123 --branch polishing💅
🔧   Creating project called `foobar123`...
Error: liquid:  --> 8:4
  |
8 | {% raw %}␊
  |    ^-^
  |
  = Unknown tag.
  with:
    requested=raw
    available tags=
    available blocks=
```

This seems to be a bug in https://github.com/ashleygwilliams/cargo-generate

**Comment by ethanfrey** [comment]
I read again, and exclude doesn't mean not copy, it means don't apply renaming. https://github.com/ashleygwilliams/cargo-generate#include--exclude

So.. my idea of using project name here is wrong. We can just fix the job name to "test" or something like that

**Comment by ethanfrey** [comment]
Closed by #8
